src/Application/Apps/Commands/CreateAppCommand.cs

```diff
@@ -13,6 +13,9 @@ public class CreateAppCommand : IRequest<Guid>
 
     [Required]
     public string StorageId { get; set; } = "";
+
+    [Required]
+    public string? CreatedBy { get; set; } = null;
```

Comment:
See if this is needed or if this is set already by the interceptor.

Reply:
We don't need the CreatedBy property on the CreateAppCommand since it's been set by the interceptor